Jenkinsfile

```
@@ -34,6 +34,18 @@ pipeline {
     }
 
     stages {
+        stage('Did you run pre-push?') {
+            steps {
+                sh '''
+                    make pre-push
+                    if [[ -z $(git status -s) ]]; then
```

This returns true if the length of `git status -s` is 0 which would be the "good" state and you would not want to fail on that. Use `-n` which will be true if the length of git status is longer than 0

```suggestion
                    if [[ -n "$(git status -s)" ]]; then
```